client/landing/subscriptions/site-list/site-list.tsx

```diff
@@ -22,6 +22,7 @@ export default function SiteList( { sites }: SiteProps ) {
 				<span className="email-frequency" role="columnheader">
 					{ translate( 'Email frequency' ) }
 				</span>
+				<span className="actions" role="columnheader" />
```

Comment:
I am a bit unclear about the class name we should define for here based on [our guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md).

I realize that we define our components outside of `client/components`. Main reason being they are specific to our entry point. Therefore, we decided to add the `subscription-manager__` prefix to all our class names. This is not strictly as per guideline, but the guideline does not take into consideration separate apps (entry points) within calypso, and what we do makes sense to me.

For this specific example of `actions` class name it still feels to me like it should be prepended with the component name it belongs to, i.e. `subscription-manager__site-list__actions`. What do you think @phcp ? 

I realize you were going by what is here already. All the other already defined cells, they all have such class names without the prefix applied. Maybe we can merge this in as is, and then have a separate PR to address our class names, to make sure they all are prefixed with their component name. What do you think?

Reply:
Yes, totally agree. I just kept the same format as the other classes on the component. But sure, I can fix it on a follow-up PR.